webpack/assets/javascripts/react_app/routes/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Switch, Route } from 'react-router-dom';
+
+import { routes } from './routes';
+
+const AppSwitcher = () => (
+  <Switch>
+    {routes.map(({ render, ...props }) => (
+      <Route
+        {...props}
+        key={props.path}
+        render={renderProps => {
+          const railsContainer = document.getElementById('rails-app-content');
+          if (railsContainer) railsContainer.remove();
```

Comment:
just wondering if this is the right "react way"

Reply:
quoting @sharvit 

> It's not trivial, you can't render rails content into react component and you can only control the rails content by using the element.